packages/ui/src/components/DatePicker/PreviousDateRangePicker.tsx

```diff
@@ -60,6 +72,64 @@ function toDateTimeString(date: Date, showYear: boolean) {
 	return date.toLocaleDateString('en-us', options)
 }
 
+/**
+ * Takes in a time string in the format HH:mm aa or HH:mmaa and returns	an object with the hour, minute, time of day, and 24 hour time
+ * @param timeString
+ * @returns {	hour: number, minute: number, timeOfDay: string, hour24: number}
+ */
+export const getTimeInfo = (timeString: string) => {
+	const date = moment(timeString, TIME_INPUT_FORMAT)
+	return {
+		hour: date.format('HH'),
+		minute: date.minute(),
+		timeOfDay: date.format('a'),
+		hour24: date.hour(),
+	}
+}
+
+/**
+ * Takes in a date and a time string in the format HH:mm aa or HH:mmaa and returns a new date with the time set
+ * @param date Date
+ * @param hour number
+ * @param minute number
+ * @returns new date	with time set
+ */
+const setTimeOnDate = (date: Date, hour: number, minute: number) => {
+	const newDate = new Date(date)
+
+	newDate.setHours(hour)
+	newDate.setMinutes(minute)
+
+	return newDate
+}
+
+/**
+ * Converts a Date object to a string in the format "HH:mm aa".
+ *
+ * @param {Date} date - The Date object to convert.
+ * @returns {string} A string in the format "HH:mm aa".
+ */
+const getTimeStringFromDate = (date: Date): string => {
+	if (!date) {
+		return '12:00 AM'
+	}
+
+	return moment(date).format(TIME_DISPLAY_FORMAT)
```

Comment:
this placeholder text doesn't make sense now. 15:44 PM isn't a valid time - this should be 3:44 PM

![image](https://user-images.githubusercontent.com/1351531/234718621-83373dd1-f9f3-4fd6-9bfa-4b8e852b444b.png)

Reply:
Hey @Vadman97 it's looking good now, fixed it and updated the placeholder to display the time in 12 hours format.

Reply:
thanks, looks good!